config/maintainers.json

```
@@ -20,6 +20,16 @@
       "link_url": null,
       "avatar_url": null,
       "bio": "Front-end web developer with years of experience using JavaScript, but learning a functional approach to web development using PureScript and Haskell"
+    },
+    {
+      "github_username": "ferhatelmas",
+      "alumnus": false,
+      "show_on_website": true,
```

```suggestion
      "show_on_website": true,
```

cc @ErikSchierboom see if ya want @ferhatelmas as contributing or senior

---

I'll defer to @iHiD :D